openslides/agenda/static/js/agenda/base.js

```diff
@@ -21,6 +21,30 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
     }
 ])
 
+.factory('AgendaUpdate',[
+    'DS',
+    'Agenda',
+    function(DS, Agenda){
+        return DS.defineResource({
```

Comment:
Can you please explain why you need a DS resource not just a factory that contains the `saveChanges` function?

Like:
```javascript
.factory('AgendaUpdate',[
    'Agenda',
    function (Agenda) {
        return {
            saveChanges: function (item_id, changes) {
                Agenda.find(item_id).then(function(item) {
                    var something = false;
                    _.each(changes, function(change) {
                        if (change.value !== item[change.key]) {
                            item[change.key] = change.value;
                            something = true;
                        }
                    });
                    if (something === true) {
                        Agenda.save(item);
                    }
                });
            }
        };
    }
])

```